apps/Launcher/qml/style.js

```diff
@@ -33,6 +33,7 @@ var fileBrowserDiscreteTextColor = "#B0B0B0"
 var fileBrowserBlinkColor = "lightblue"
 var fileBrowserItemSizeRel = 0.2
 var fileBrowserTextSizeRelToItem = 0.09
+var fileBrowserGridSectionTextRelSize = 0.35
```

Comment:
there were more numbers in the new files, but no idea if they should be here as well